### ISSUE TITLE
Improve Compliance to Web Content Accessibility Guidelines (WCAG)

### ISSUE THREAD

**Issue by bl-aire** [unicef/inventory-hugo-theme#101]
# Summary

Improve user experience by making site more compliant to W3C's Web Content Accessibility Guidelines.

# Background 

When building projects, it is easy to focus on just having a bug free code that functions as it should, forgetting that actual people use these products and each of these users have their own need that has to be met.

To test accessibility of the site, I used WAVE and axe tools to check its conformance with W3C's guidelines and found out there are bits of the project that can be improved.

A few of them include but aren't restricted to the following:

1. Inadequate contrast 
2. Missing form labels 
3. Page content not contained by landmarks etc

These are shown below:

![image](https://user-images.githubusercontent.com/84882370/163219781-f1319359-8a59-41b9-a069-8e9cdd31004d.png)

![image](https://user-images.githubusercontent.com/84882370/163219879-21ea7627-0417-4998-aa51-782725c1781e.png)

The improvement might be as easy as adding an alt attribute, using <strong> instead of <bold> so screen readers can easily read words making proper emphasis where needed for people who suffer visual impairment, using appropriate labels when creating forms, making sure there’s enough contrast between elements and may seem insignificant but a person’s life is being affected directly and so it matters.

# Detail

I am proposing these bits of corrections that have been pointed out are made and one can approach this by making changes and checking intermittently for progress using accessibility testing tools (WAVE, axe etc).

To get a better understanding of what the guidelines are, research is needed so one understands why changes are being made and the knowledge gained can be used when building subsequent web based products.

# Outcome

The site will become more accessible as it is easier for people with special needs to use and generally the user - experience is better.

**Comment by bl-aire** [comment]
Hi @jwflory,

I would love to work on this.

**Comment by jwflory** [comment]
Hi @bl-aire, I'm sorry I did not respond to this sooner. The feedback you outlined here is excellent, and I agree that we should prioritize accessibility improvements to the Inventory theme.

Are you still interested in working on this issue?

**Comment by bl-aire** [comment]
> Hi @bl-aire, I'm sorry I did not respond to this sooner. The feedback you outlined here is excellent, and I agree that we should prioritize accessibility improvements to the Inventory theme.
> 
> 
> 
> Are you still interested in working on this issue?

Thank you.

I am still interested in working on it. And I have taken note of the review made for PR #99 and #100. I will make corrections and improvements on both.

**Comment by jwflory** [comment]
Thanks, @bl-aire! Looking forward to your continued contributions. ✨
